// _@vuepress_core@1.2.0@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "C:\\Users\\Administrator\\Desktop\\Tiny-blog\\node_modules\\_@vuepress_core@1.2.0@@vuepress\\core\\lib\\client\\components\\GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-2472ecc2",
    path: "/blog/jwt02.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-2472ecc2").then(next)
    },
  },
  {
    name: "v-47cae86e",
    path: "/blog/redis-sentinel.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-47cae86e").then(next)
    },
  },
  {
    name: "v-1df5f824",
    path: "/blog/spring-boot-annotation.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-1df5f824").then(next)
    },
  },
  {
    name: "v-0886ab40",
    path: "/blog/nginx-cdn.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-0886ab40").then(next)
    },
  },
  {
    name: "v-d800a824",
    path: "/blog/spring-boot-pagehelper&&tkmybatis2.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-d800a824").then(next)
    },
  },
  {
    name: "v-4b79c9cc",
    path: "/blog/spring-boot-pagehelper&&tkmybatis.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-4b79c9cc").then(next)
    },
  },
  {
    name: "v-590a9ace",
    path: "/blog/spring-boot-static.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-590a9ace").then(next)
    },
  },
  {
    name: "v-cf0ecdd8",
    path: "/blog/spring-boot-thymeleaf.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-cf0ecdd8").then(next)
    },
  },
  {
    name: "v-3b1c3dc4",
    path: "/blog/jwt01.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-3b1c3dc4").then(next)
    },
  },
  {
    name: "v-67eb4b0e",
    path: "/blog/spring-boot-swagger2.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-67eb4b0e").then(next)
    },
  },
  {
    name: "v-7f36652e",
    path: "/blog/spring-boot-validation.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-7f36652e").then(next)
    },
  },
  {
    name: "v-34506874",
    path: "/blog/spring-boot-start.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("post", "v-34506874").then(next)
    },
  },
  {
    name: "v-48cdc28c",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-48cdc28c").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
